use-cases/messaging.md

```diff
@@ -0,0 +1,156 @@
+## Messaging
+
+Enable guests to send messages to the enterprise and receive replies, by connecting your chatbot or instant messaging app into the customer messaging system of __Mews Operations__.
+You can do this through the API with the [Messages](../operations/README.md#messages) family of API operations, in conjunction with the `MessageAdded` [General Webhook](../webhooks/wh-general.md).
```

Comment:
`Messages family` ... is it a common thing how to name API? I had to read it several times to get what it means. ... Just question, if it's standard, then ok 😊

Reply:
I am just using 'family' to mean 'group' or 'set', but if you had to read it several times to get the meaning, then it's not doing it's job very well and I should re-write it!